global-script.lua

```diff
@@ -3198,6 +3200,8 @@ function swapPlayerAreas(a, b)
     tableSwap(elementScanZones)
     updateBlock(a)
     updateBlock(b)
+
+    print(a .. " traded places with " .. b .. ".")
```

Comment:
can we switch this to used broadcast? maybe sent to both of the players in question?

Reply:
We could.  However, there's one major advantage to sending it to everyone as it allows the people at the table to self-police if someone obnoxiously keeps swapping colors.

If Red keeps swapping places repeatedly with Blue, the messaging currently tells everyone both that the swap happened and that Red is the person initiating the swap.  I'm imagining the dysfunctional player group conversation of "Red, keep doing that!"  "I'm not doing it, blue is!" -- this messaging makes it clear(ish) who is doing it.

Reply:
sure that's fine it can be sent to everyone